Domains/Grids/grids.remarks.md

```diff
@@ -5,17 +5,17 @@ remarksTarget: Grids.ecschema.md
 
 # Grids
 
-This schema contains class definitions for grids.
+## Overview
 
-These are used to build structural,spaceplanning and other grids. A `Grid` is a collection of GridSurfaces. Every `GridSurface` has a `GridAxis`, which is currently primarily used for grouping surfaces into subgroups. intersection of GridSurfaces may create a `GridCurve` (driven by `GridCurveBundle` element).
+Contains classes that support both simple Grids (which are sets of curves defined in a particular plane) and GridSystems (which contain sets of surfaces organized into axes which can be used to instruct iTwin.js to generate simple Grids). 
 
-<u>Schema:</u>
+A `Grid` is equivalent to an [IfcGrid](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC4-voting/HTML/link/ifcgrid.htm). IFC has no equivalent of a GridSystem.
 
-```xml
-<ECSchema schemaName="Grids" alias="grids" version="01.00" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.1">
-    <ECSchemaReference name="BisCore" version="01.00.00" alias="bis" />
-    <ECSchemaReference name="AecUnits" version="01.00.00" alias="AECU" />
-```
+Classes in this schema are used to build Structural, Space-planning and other `GridSystem`s and `Grid`s. A `GridSystem` is a collection of GridSurfaces. Every `GridSurface` is associated to a `GridAxis`, which is currently primarily used for grouping surfaces into subgroups. The intersection of `GridSurface`s may be used to create a `GridCurve`. The constructed `GridCurve` should then inherit the userlabel of a `GridLabelRecord` (if present) of one of the originating `GridSurface`s. Some viewing application would show the information of `GridLabelRecord` as a dynamic annotation, similar to [Grid Labels](https://docs.bentley.com/LiveContent/web/OpenBuildings%20Designer%20Help-v5/en/GUID-5965684D-5CA4-BF92-3406-A43B5542ED30.html) in `OpenBuildings Designer`. 
```

Comment:
"_which is **currently** primarily_" - Is the word _currently_ really unnecessary? It is giving me the impression the current situation is temporary or a workaround for something better.

Reply:
it's not necessary.

the GridAxis in a future version of this schema could carry information necessary to create GridLabelRecords on newly added GridSurfaces. (labeling format i.e. 1,2,3 or A,B,C or "Level 1", "Level 2" etc)